.travis.yml

```diff
@@ -30,53 +23,47 @@ cache:
   directories:
     - ~/.cache # Cypress installed into ~/.cache/Cypress
 
-# Build stages
-jobs:
-  include:
-    # Stage: Tests
-    - stage: test
-      script: npm run test-ci
-
-    # Stage: Build & deploy (only on master)
-    - stage: build
-      script: npm run build-production
-      after_success:
-        - npm prune --production > /dev/null 2>&1
-        - sh deploy/store_deploy_info.sh
-        - zip -qr latest * -x .\* -x "README.md" -x assets/\* -x "gulpfile.js"
-        - mkdir -p dpl_cd_upload
-        - mv latest.zip dpl_cd_upload/build-$TRAVIS_BUILD_NUMBER.zip
-      deploy:
-        - provider: s3
-          region: eu-west-1
-          bucket: blf-assets
-          access_key_id: $AWS_ACCESS_KEY_ID
-          secret_access_key: $AWS_SECRET_ACCESS_KEY
-          local_dir: public
-          upload-dir: assets
-          acl: public_read
-          cache_control: 'max-age=31536000'
-          skip_cleanup: true
-        - provider: s3
-          region: eu-west-2
-          bucket: blf-deployments
-          access_key_id: $AWS_ACCESS_KEY_ID
-          secret_access_key: $AWS_SECRET_ACCESS_KEY
-          local_dir: dpl_cd_upload
-          skip_cleanup: true
-          on:
-            repo: biglotteryfund/blf-alpha
-        - provider: codedeploy
-          region: eu-west-2
-          access_key_id: $AWS_ACCESS_KEY_ID
-          secret_access_key: $AWS_SECRET_ACCESS_KEY
-          bucket: blf-deployments
-          key: build-$TRAVIS_BUILD_NUMBER.zip
-          bundle_type: zip
-          application: BLF_WWW
-          deployment_group: Test_Fleet
-          on:
-            tags: false
+script:
+  - npm run build-production
```

Comment:
Small note here: Arguably unit tests (`npm test`) should come before the build but we have a single unit test (RC email tests) which require a build due to the way they assert email templates. This might be an appropriate trade-off but feels like a bit of smell, something to revisit.